Calculator SwiftUIUITests/Tests/CalculatorTest.swift

```diff
@@ -1,3 +1,4 @@
+
 import XCTest
 
 final class Calculator_UITest: BaseSetupTest{
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/DawidBastek/UITestsForCalculator/blob/a0eb0715c6ef158aa10d2e3bb9e0540ca70b993d/Calculator SwiftUIUITests/Tests/CalculatorTest.swift#L4"></span>Type name should only contain alphanumeric characters: 'Calculator_UITest'<br />
<code>type_name</code> <code>CalculatorTest.swift:4</code></td>
    </tr>
  </tbody>
</table>